lambda_functions/ml_service/template.yaml

```diff
@@ -68,7 +68,7 @@ Resources:
         Echo:
           Type: Api # More info about API Event Source: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#api
           Properties:
-            Path: /factchecks/{item_id}
+            Path: /tags/{item_id}
```

Comment:
I still dislike the idea of the `item_id` being a pathParameter.

Strictly following the REST standard, the path is usually meant to identify a document or a collection of documents (in this case a tag or multiple tags). https://restfulapi.net/resource-naming/

If we want to follow the REST standard, I would propose one of the following path templates:

- /items/{item_id}/tags
- /tags?item_id={item_id}

The later would make it necessary to also change the GetTags function to read the item_id from the queryStringParameters. (see [create_review.py](https://github.com/DetektivKollektiv/python-lambda-functions/blob/develop/lambda_functions/review_service/create_review.py#L31) for an example)

Reply:
@PotSchmoell any feelings?

Reply:
Thanks for the link :-) I give the first option a try

Reply:
I think I would prefer query parameters (option 2). 
They are also rather easy to implement. Check "detektivkollektiv-review_api-get_review_question" in the review service for reference.